crates/ruff/src/rules/pylint/snapshots/ruff__rules__pylint__tests__PLR5501_collapsible_else_if.py.snap

```diff
@@ -1,26 +1,39 @@
 ---
 source: crates/ruff/src/rules/pylint/mod.rs
 ---
-collapsible_else_if.py:38:9: PLR5501 Use `elif` instead of `else` then `if`, to reduce indentation
+collapsible_else_if.py:37:5: PLR5501 Use `elif` instead of `else` then `if`, to reduce indentation
    |
+35 |       if 1:
 36 |           pass
 37 |       else:
-38 |           if 2:
-   |  _________^
-39 | |             pass
-   | |________________^ PLR5501
+   |  _____^
+38 | |         if 2:
```

Comment:
Nit: Should the range point to the end of the `if` condition?

Reply:
i made it that way because it now spans the region that you have to replace with an elif, i'm fine to change the ranges later